spec/requests/v1/songs_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe 'songs endpoints' do
+  describe 'GET /index' do
+    it 'returns JSON for all songs' do
```

Comment:
FYI - often, when having multiple test cases, it can be helpful to wrap them in a 

```
context 'when there are several songs` do
  it 'returns JSON for all songs' do
    ...
  end
end
```

and

```
context `when there are no songs` do
...
end
```

To give ... context ... for the scenario that the expectation is validating.